### PR TITLE
backport qgis-server tutorial

### PR DESCRIPTION
**Pull request by tudorbarascu**

### CODE REVIEW

File: source/docs/training_manual/qgis_server/install.rst [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+This is a good thing, it tells you we're on the right track as the server is
+saying that we haven't asked for a supported service. We'll see later on
+how to make WMS requests.
+
+|FA| HTTP Server Configuration
+-------------------------------------------------------------------------------
+
+In order to access on the installed QGIS server from an Internet Browser we
+need to use a HTTP server.
```
Comment (yjacolin) [inline source/docs/training_manual/qgis_server/install.rst]: `an HTTP server`?

File: source/docs/training_manual/qgis_server/install.rst [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+In this lesson we're going to use the
+`Apache HTTP server <http://httpd.apache.org>`_, colloquially called Apache.
+
+First we need to install Apache by running the following command in a terminal:
+``sudo apt-get install apache2 libapache2-mod-fcgid``.
+
+Let's create a file called :file:`qgisplatform.demo.conf` in that directory
```
Comment (yjacolin) [inline source/docs/training_manual/qgis_server/install.rst]: which directory?

File: source/docs/training_manual/qgis_server/install.rst [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+ sudo chown www-data:www-data /logs
+
+ mkdir /home/qgis/qgisserverdb
+ sudo chown www-data:www-data /home/qgis/qgisserverdb
+
+.. note::
+
+ ``www-data`` is the Apache user on Debian based systems and we need Apache to have access to
+ those locations or files.
+ The ``chown www-data...`` commands changes the owner of the respective directories/files
```
Comment (yjacolin) [inline source/docs/training_manual/qgis_server/install.rst]: `command` (single)

File: source/docs/training_manual/qgis_server/install.rst [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+.. note::
+
+ curl can be installed with ``sudo apt-get install curl -y``.
+
+Apache is now configured.
+
+|moderate| |FA| Create another virtual host
+-------------------------------------------------------------------------------
+
```
Comment (yjacolin) [inline source/docs/training_manual/qgis_server/install.rst]: could you explain why the user need to create another vhost? Sorry I am pretty sure we already discuss this in your previus PR but it worths to write it here. What's your opinion?

File: source/docs/training_manual/qgis_server/wms.rst [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+
+.. note::
+
+ * If you look at the logs in the following sections you should get a better
+   understanding on what's happening.
+ * By restarting Apache while looking in the QGIS Server log you can find
+   some extra pointers on how things work.
+
+GetMap requests
+-------------------------------------------------------------------------------
```
Comment (yjacolin) [inline source/docs/training_manual/qgis_server/wms.rst]: just `----------------` is enough :)

File: source/docs/training_manual/qgis_server/install.rst (hunk 4 above)
Comment (tudorbarascu) [inline source/docs/training_manual/qgis_server/install.rst]: To ensure the concept of virtualhost is understood. He doesn't need to create it.

**Pull request merged**